test/extract-stream.js

```diff
@@ -0,0 +1,23 @@
+'use strict'
+
+var fs = require('fs')
+var npmlog = require('npmlog')
+var pipe = require('mississippi').pipe
+var test = require('tap').test
+
+require('./util/test-dir')(__filename)
+
+var extractStream = require('../lib/extract-stream')
+
+npmlog.level = process.env.LOGLEVEL || 'silent'
+var OPTS = {
+  log: npmlog
+}
+
+test('basic extraction')
```

Comment:
Not gonna forget. These are *very visible* if you run `npm t`. Also, #48 is the PR adding these tests (and `registry.tarball.js`)